### ISSUE TITLE
When Export Python Code,if contains Boolean Value ,such as 'fasle',the export code is wrong.

### ISSUE THREAD

**Issue by wcc526** [mitmproxy/mitmproxy#1187]
##### Steps to reproduce the problem:

1. `curl -v -X POST --data "@data.json" http://example.com -x 127.0.0.1:8080`
  the data.json is like this,

  ```
 {
   "abc": false,
 }
  ```
2. Export python code
3. the python code is like this

 ```
json = {
    "abc": false,
}

response = requests.request(
    method='POST',
    url=url,
    headers=headers,
    json=json,
)

print(response.text)
 ```

##### What is the expected behavior?

 ```
json = {
    "abc": 'false',
}

response = requests.request(
    method='POST',
    url=url,
    headers=headers,
    json=json,
)

print(response.text)
  ```
##### What went wrong?
Maybe the export code has bug

##### Any other comments? What have you tried so far?

---

Mitmproxy Version:
Operating System:

**Comment by MatthewShao** [comment]
In json, `false` is a boolean value, when we output the python code, it should be a negative boolean value, which is `False` in python, instead of the string `'false'`.
So I think the correct output should be like this:
```
json = {
    "abc": False,
}

response = requests.request(
    method='POST',
    url=url,
    headers=headers,
    json=json,
)

print(response.text)
```

**Comment by dufferzafar** [comment]
I'm not sure what semantics of curl are, but I just tested this out with `[httpie](https://github.com/jkbrzt/httpie)` `http POST httpbin.org/post < data.json --proxy=http:http://localhost:8080`

where `data.json` contains:

```json
{
    "data": false
}
```

and when I exported this request as Python code, the correct stuff was copied:

```python
import requests

url = 'http://httpbin.org/post'

headers = {
    'Host': 'httpbin.org',
    'Content-Length': '15',
    'Accept-Encoding': 'gzip, deflate',
    'Accept': 'application/json',
    'User-Agent': 'HTTPie/0.9.3',
    'Connection': 'keep-alive',
    'Content-Type': 'application/json',
}


json = {
    'data': False,
}


response = requests.request(
    method='POST',
    url=url,
    headers=headers,
    json=json,
)

print(response.text)
```

**Comment by dufferzafar** [comment]
I figured out why this is happening. `JSON` detection works by checking out the contenty-type header which is not set in the case of the `curl` command but is automatically set by `httpie`.

I tested out the exporting of these two commands:

`curl -v -X POST --data "@data.json" http://example.com -x 127.0.0.1:8080`

`curl -v -X POST --data "@data.json" http://example.com -x 127.0.0.1:8080 -H "Content-Type: application/json"`

and in the first case, the data isn't parsed as json while in the second case it is and `false` is properly handled.
